packages/vue-dot/src/patterns/FormBuilder/mixins/tests/formBuilderCore.spec.ts

```diff
@@ -228,4 +234,30 @@ describe('formBuilderCore', () => {
 			expect(wrapper.emitted().change[0]).toEqual([updatedTestForm]);
 		});
 	});
+
+	it('don\'t emits refresh event', async() => {
```

Comment:
```suggestion
	it('doesn\'t emits refresh event', async() => {
```
Et il n'émet pas d'événement refresh lorsque quoi ? :thinking: